packages/ui/src/components/createEvent.js

```diff
@@ -0,0 +1,328 @@
+/**
+ * @author Ragasudha Aradhyula
+ * @name Create Event
+ * @desc This component renders the create event form page
+ */
+import React, { Component} from 'react';
+import { Card } from 'material-ui/Card';
+import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import DatePicker from 'material-ui/DatePicker';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+import {RadioButton, RadioButtonGroup} from 'material-ui/RadioButton';
+import { Route } from 'react-router-dom';
+import History from '../history';
+import { INVALID_USER } from "../constants/actions";
+
+const participants = [
+  <MenuItem key={1} value={1} primaryText="Organizer" />,
+  <MenuItem key={2} value={2} primaryText="Volunteer" />
```

Comment:
Don't hardcode the values here, get just the primaryText as an array from constants and inside a loop render it.
Participants = ['Organizer', 'Volunteer'] => This should come from constant

renderParticipants = () => {
Participants.map((participant, index) => {
return <MenuItem key={index} value={index} primaryText={participant} />
})
}

Something of this sort

Reply:
These values will be removed once the API is ready. Madhulika is working on it